haas/client/user.py

```diff
@@ -0,0 +1,46 @@
+import json
+from haas.client.base import ClientBase
+from haas.client import errors
+
+
+class User(ClientBase):
+    """ Consists of calls to query and manipulate users related
+    objects and relations.
+    """
+
+    def create(self, username, password, privilege):
+        """Create a user <username> with password <password>.
+    <privilege> may by either "admin" or "regular", and determines whether a
+    user is authorized for administrative privileges
+    """
```

Comment:
1. Blank line between the one-line summary and the rest of the docstring, per pep257.
2. indent the rest of the docstring to match the first line.

(1) applies to many ohter docstrings in this patch; please go through and fix it everywhere.